model/event/subscriptions.go

```diff
@@ -248,10 +249,34 @@ func (s *Subscription) Validate() error {
 	if !IsValidOwnerType(string(s.OwnerType)) {
 		catcher.Add(errors.Errorf("%s is not a valid owner type", s.OwnerType))
 	}
+	catcher.Add(s.runCustomValidation())
 	catcher.Add(s.Subscriber.Validate())
 	return catcher.Resolve()
 }
 
+func (s *Subscription) runCustomValidation() error {
+	catcher := grip.NewBasicCatcher()
+
+	if taskDurationVal, ok := s.TriggerData["task_duration_secs"]; ok {
```

Comment:
You define this as a constant in model. Define it as a constant once, and then use that?